core/tests/protractor_desktop/adminTabFeatures.js

```diff
@@ -57,28 +57,6 @@ describe('Admin Page', function() {
     await users.logout();
   });
 
-  it('should run,verify and stop one-off jobs', async function() {
```

Comment:
Why are we removing this test?

Reply:
I was in a meeting with Sean, and he suggested that we take this test out since it was failing & blocking my PR. He also mentioned that the jobs page would be getting revamped, so the test would need to be completely rewritten.

Reply:
/cc @brianrodri FYI

Reply:
(also @vojtechjelinek)

Reply:
SGTM.

Reply:
Cool, thanks for explaining!

Reply:
Do we need to track that the test gets added back after the rewrite? Note sure.

Reply:
Yep we'll have an equivalent Apache Beam e2e test and will work hard to make sure it isn't flaky